src/Behat/Gherkin/Filter/LineFilter.php

```diff
@@ -119,4 +100,25 @@ public function filterFeature(FeatureNode $feature)
             $feature->getLine()
         );
     }
+
+    /**
+     * Checks if scenario or outline matches specified filter.
+     *
+     * @param ScenarioInterface $scenario Scenario or Outline node instance
+     *
+     * @return Boolean
+     */
+    public function isScenarioMatch(ScenarioInterface $scenario)
```

Comment:
please don't move method around. It makes the review harder (I know need to check manually if the moved code is the same) and breaks ``git blame`` without any benefit